api/CSSPropertyRule.json

```diff
@@ -28,7 +28,7 @@
           "webview_android": "mirror"
         },
         "status": {
-          "experimental": false,
+          "experimental": true,
```

Comment:
Also in Safari https://github.com/WebKit/WebKit/blob/main/Source/WebCore/css/CSSPropertyRule.idl